website/pages/docs/agent/config-entries/service-intentions.mdx

```diff
@@ -0,0 +1,331 @@
+  [endpoints](/api-docs/connect/intentions#read-specific-intention-by-id) to
+  continue to function for a period of time after ugprading to 1.9.0.
+  TODO(link-do-long-form-docs)
+
+### `IntentionPermission`
+
+- `Action` `(string: <required>)` - This is one of "allow" or "deny" for
+  the action that should be taken if this permission matches a request.
+
+- `HTTP` `(IntentionHTTPPermission: <required>)` - A set of
+  [http-specific authorization criteria](#intentionhttppermission).
+
+### `IntentionHTTPPermission`
+
+- `PathExact` `(string: "")` - Exact path to match on the HTTP request path.
+
+  At most only one of `PathExact`, `PathPrefix`, or `PathRegex` may be configured.
+
+- `PathPrefix` `(string: "")` - Path prefix to match on the HTTP request path.
+
+  At most only one of `PathExact`, `PathPrefix`, or `PathRegex` may be configured.
+
+- `PathRegex` `(string: "")` - Regular expression to match on the HTTP
+  request path.
+
+  The syntax is [described below](#regular-expression-syntax).
+
+  At most only one of `PathExact`, `PathPrefix`, or `PathRegex` may be configured.
+
+- `Methods` `(array<string>)` - A list of HTTP methods for which this match
+  applies. If unspecified all http methods are matched.
+  If provided the names must be [valid](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods).
+
+- `Header` `(array<IntentionHTTPHeaderPermission>)` - A set of criteria that can
+  match on HTTP request headers. If more than one is configured all must match
+  for the overall match to apply.
+
+  - `Name` `(string: <required>)` - Name of the header to match.
+
+  - `Present` `(bool: false)` - Match if the header with the given name is
+    present with any value.
+
+    At most only one of `Exact`, `Prefix`, `Suffix`, `Regex`, or `Present`
+    may be configured.
+
+  - `Exact` `(string: "")` - Match if the header with the given name is this
+    value.
+
+    At most only one of `Exact`, `Prefix`, `Suffix`, `Regex`, or `Present`
+    may be configured.
+
+  - `Prefix` `(string: "")` - Match if the header with the given name has
+    this prefix.
+
+    At most only one of `Exact`, `Prefix`, `Suffix`, `Regex`, or `Present`
+    may be configured.
+
+  - `Suffix` `(string: "")` - Match if the header with the given name has
+    this suffix.
+
+    At most only one of `Exact`, `Prefix`, `Suffix`, `Regex`, or `Present`
+    may be configured.
+
+  - `Regex` `(string: "")` - Match if the header with the given name matches
+    this pattern.
+
+    The syntax is [described below](#regular-expression-syntax).
+
+    At most only one of `Exact`, `Prefix`, `Suffix`, `Regex`, or `Present`
+    may be configured.
+
+  - `Invert` `(bool: false)` - Inverts the logic of the match.
+
+## ACLs
+
+Configuration entries may be protected by
+[ACLs](https://learn.hashicorp.com/tutorials/consul/access-control-setup-production).
+
+Reading a `service-intentions` config entry requires `intentions:read` on itself.
```

Comment:
```suggestion
Reading a `service-intentions` config entry requires `intentions:read` on the destination service.
```

Reply:
If you agree with this suggestion, should we also consider changing this on the docs for the other L7 config entries?

Reply:
How about just the more generic "on its name", since that is unambiguous and the same across all of them.

Reply:
:+1: